smoke-tests/apps/TelemetryProcessors/src/smokeTest/resources/applicationinsights.json

```
@@ -89,9 +89,9 @@
         "type": "attribute",
         "actions": [
           {
-            "key": "httpPath",
-            "pattern": "(?<httpPath>.+)",
-            "replace": "*${httpPath}*",
+            "key": "http.url",
+            "pattern": "(?<userGroupName>http:\\/\\/localhost:\\d+\\/TelemetryProcessors\\/user\\/)\\d+",
+            "replace": "${userGroupName}***",
```

I think this should work also
```suggestion
            "key": "http.url",
            "pattern": "/user\\/)\\d+",
            "replace": "/user/****",
```